tests/CMakeLists.txt

```diff
@@ -156,4 +180,17 @@ target_link_libraries(
         OpenImageIO::OpenImageIO
 )
 
+setup_test_coverage(Test_UsageTimer)
 add_test ( NAME Test_UsageTimer COMMAND Test_UsageTimer )
+
+
+
+################################################################################
+# Coverage report generation
+if( ENABLE_COVERAGE AND COVERAGE_SUPPORTED )
+    # Include config_tests subdirectories
+    add_subdirectory(config_tests/core)
+    add_subdirectory(config_tests/util)
```

Comment:
this basically makes sure that congi_tests are run as part of coverage. So if something is only through these tests should technically add coverage. Thought enabling disabling them I didn't see change in coverage.